custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -58,10 +69,10 @@ class ScenarioSimilarities(
             editProbability = 0.00f, // 0.10f if we can mutate data
             commentProbability = 0.00f, // 0.04f if we can mutate data
             linkIssueProbability = 0.00f, // 0.10f if we can mutate data
-            attachScreenShotProbability = 0.05f,
-            changeAssigneeProbability = 0.00f,
-            mentionUserProbability = 0.00f,
-            transitionProbability = 0.00f,
+            attachScreenShotProbability = 0.00f, // 0.04f if we can mutate data
```

Comment:
Pro-tip: instead of switching these back and forth, you can control those via a single boolean and switch just the boolean.